sched/irq/irq_csection.c

```diff
@@ -345,6 +345,26 @@ irqstate_t enter_critical_section(void)
         }
       else
         {
+          /* CHECK IRQ
+           *
+           * After the OS starts, in the thread context,
+           * we must ensure that the interrupt is not masked
+           * when first enters the critical section.
+           * Otherwise, at this time when another CPU issues a pause
+           * interrupt to the current CPU, it can lead to a deadlock.
+           *
+           * During the vela os startup phase, interrupts are masked until
```

Comment:
```suggestion
           * During the OS startup phase, interrupts are masked until
```
Do not include any specific platform comments in NuttX Community FYI @acassis